example/config/managedk8s.yaml

```
@@ -1,281 +1,282 @@
-    #     - matchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    #       acceptedTraffic:
-    #         ingress: true
-    #         egress: true
-    # - ruleID: "2001"
-    #   args:
-    #     acceptedPods:
-    #     - matchLabels:
-    #         foo: bar
-    #       namespaceMatchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    # - ruleID: "2002"
-    #   args:
-    #     acceptedStorageClasses:
-    #     - labelSelector:
-    #         matchLabels:
-    #           foo: bar
-    #       justification: "justification"
-    # - ruleID: "2003"
-    #   args:
-    #     acceptedPods:
-    #     - matchLabels:
-    #         foo: bar
-    #       namespaceMatchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    #       volumeNames:
-    #       - "volume-a"
-    #       - "volume-b"
-    #     - matchLabels:
-    #         foo: baz
-    #       namespaceMatchLabels:
-    #         foo: baz
-    #       justification: "justification"
-    #       volumeNames:
-    #       - "*" # a wildcard can be used to match against all volumes in an accepted pod
-    # - ruleID: "2004"
-    #   args:
-    #     acceptedServices:
-    #     - matchLabels:
-    #         foo: bar
-    #       namespaceMatchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    # - ruleID: "2005"
-    #   args:
-    #     allowedImages:
-    #     - prefix: "example.foo.repository/organisation/releases/"
-    # - ruleID: "2006"
-    #   args:
-    #     acceptedRoles:
-    #     - matchLabels:
-    #         foo: bar
-    #       namespaceMatchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    #     acceptedClusterRoles:
-    #     - matchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    # - ruleID: "2007"
-    #   args:
-    #     acceptedRoles:
-    #     - matchLabels:
-    #         foo: bar
-    #       namespaceMatchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    #     acceptedClusterRoles:
-    #     - matchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    # - ruleID: "2008"
-    #   args:
-    #     acceptedPods:
-    #     - matchLabels:
-    #         foo: bar
-    #       namespaceMatchLabels:
-    #         foo: bar
-    #       justification: "justification"
-    #       volumeNames:
-    #       - "volume-a"
-    #       - "volume-b"
-    #     - matchLabels:
-    #         foo: baz
-    #       namespaceMatchLabels:
-    #         foo: baz
-    #       justification: "justification"
-    #       volumeNames:
-    #       - "*" # a wildcard can be used to match against all volumes in an accepted pod
+providers: # contains information about known providers
+  - id: managedk8s # unique provider identifier
+    name: "Managed Kubernetes" # user friendly name of the provider
+    metadata:
+      foo: bar
+    args:
```

```suggestion
providers:                   # contains information about known providers
- id: managedk8s             # unique provider identifier
  name: "Managed Kubernetes" # user friendly name of the provider
  metadata:
    foo: bar
  args:
```

Please do not change the original indentation (up to line 279)